packages/wix-ui-core/src/components/address-input/AddressInput.tsx

```diff
@@ -41,6 +42,8 @@ export type AddressInputProps = Pick<
   'data-hook'?: string;
   /** Maps client, should implement autocomplete, geocode and placeDetails methods */
   Client: MapsClientConstructor;
+  /** client identifier - GoogleMaps or Wix's Atlas */
+  providerName?: string;
```

Comment:
```suggestion
  providerName?: 'google' | 'atlas';
```
You should also provide a default value for this in defaultProps.
I suggest using  'google' as default, because in order to use 'atlas' you must pass `instance` prop.
So using Atlas by default would be a breaking change for people already using the component

Reply:
I agree, adding